.github/scripts/validate_binaries.sh

```diff
@@ -80,6 +74,12 @@ else
     fi
     eval $INSTALLATION
 
+    # test with numpy 1.x installation needs to happen after torch install
+    MINOR_PYTHON_VERSION=$(echo "$MATRIX_PYTHON_VERSION" | cut -d . -f 2)
+    if [[ ${MINOR_PYTHON_VERSION} < 13 ]]; then
```

Comment:
how does this check react to python 3.13t?  While this will technically evaluate to the desired result of false, b/c it's a string comparison instead of a numeric one, it might be better to make the check more explicit.

Reply:
Yes It works, tested with 13t